client/src/components/Home/index.js

```diff
@@ -65,6 +66,14 @@ function Home() {
           </Box>
 
           <Box className={classes.news_box}>
+            {isAuthenticated && (
+              <Box className={classes.daily_calories}>
+                <DailyUserCalories
+                  showCard={smallScreen || largeScreen}
```

Comment:
I think one of these two variables, either `smallScreen` or `largeScreen`, will always be true.  So this conditional, `smallScreen || largeScreen`, therefore would always evaluate to true, and so your code would be a little clearer if it just said `showCard={true}`

Reply:
Yes , you are right :) , There is not to condition here 
We miss that
Thank you bro